rllib/algorithms/impala/impala_learner.py

```diff
@@ -45,34 +45,15 @@ class ImpalaHPs(LearnerHPs):
 
     rollout_frag_or_episode_len: int = None
     recurrent_seq_len: int = None
-    discount_factor: float = 0.99
-    vtrace_clip_rho_threshold: float = 1.0
-    vtrace_clip_pg_rho_threshold: float = 1.0
-    vtrace_drop_last_ts: bool = True
-    vf_loss_coeff: float = 0.5
-    entropy_coeff: float = 0.01
+    discount_factor: float = None
```

Comment:
I found it dangerous to give these parameters new default values (there should only be one place in all RLlib, where a default value for a given HP is defined, and this is in its respective AlgorithmConfig class). Everything else is duplicating information, which is a recipe for disaster.